openedx/core/djangoapps/user_authn/views/auth.py

```diff
@@ -0,0 +1,9 @@
+from common.djangoapps.util.json_request import JsonResponse
+from django.conf import settings
+
+def issue_jwks():
```

Comment:
I'm not sure about the name `issue_jwks`, but other than that, this seems like the right path and pretty close. Thanks.

Reply:
do you have any suggestions? how about `create_jwks` or `define_jwks`?

Reply:
What about something like `get_public_signing_jwks`?